servers/src/test/java/tachyon/master/block/meta/BlockContainerIdGeneratorTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.master.block.meta;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import tachyon.master.block.BlockContainerIdGenerator;
+import tachyon.proto.journal.Block;
+import tachyon.proto.journal.Journal;
+
+/**
+ * Unit tests for {@link BlockContainerIdGenerator}.
+ */
+public final class BlockContainerIdGeneratorTest {
+  /** A non-default, container id for testing. */
+  private static final long TEST_ID = 1234;
+  private BlockContainerIdGenerator mGenerator;
```

Comment:
That's a good question. There is a limit, when it is reached, but it is not obvious what we should do about it, and where it should be handled. It is not being handled in the BlockContainerIdGenerator class currently.